### ISSUE TITLE
Tech debt: Squelch e2e cypress errors

### ISSUE THREAD

**Issue by BCerki** [button-inc/digital_marketplace#115]
### Description
As a developer
I want no errors in my CI
So that I can be confident my code is working
### Tasks (How)
- [ ] Task
- [ ] Task
- [ ] Task
Alec from [here](https://github.com/button-inc/digital_marketplace/pull/112): There's probably a way to squelch these, so that should be tracked as tech debt as well.

**Comment by wenzowski** [comment]
> You can safely ignore the dbus error if your tests are executing fine.

https://github.com/cypress-io/cypress/issues/4925#issuecomment-573576303

Closing: safe to ignore.

**Comment by djangofan** [comment]
You really should fix this because is super confusing, in my case my test suite fails, but the first output messages are these ERROR logs:

```
cypress run --record --headless '--spec=cypress/integration/smoke/*.spec.js' --env configFile=cypress/config/cypress.playground.json --browser chrome --key b1f2e1.....53d9ebcf
[259:1229/221955.995612:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
[259:1229/221956.006535:ERROR:bus.cc(392)] Failed to connect to the bus: Address does not contain a colon
[259:1229/221956.006584:ERROR:bus.cc(392)] Failed to connect to the bus: Address does not contain a colon
[425:1229/221956.055877:ERROR:gpu_init.cc(441)] Passthrough is not supported, GL is swiftshader
```
AND so it is misleading me to think that is the cause of my subsequent errors.   Are you sure I can ignore??   I am running in a cypress/included 8.7.0 .  I had the same errors with cypress-included v9

NOTE: I managed to fix this issue by deleting my /var docker folder and then completely re-installing docker on my Mac.
